squidb/src/com/yahoo/squidb/utility/SquidUtilities.java

```diff
@@ -211,7 +205,6 @@ private static void copyStream(InputStream source, OutputStream dest) throws IOE
             }
 
             dest.write(buffer, 0, bytes);
-            dest.flush();
```

Comment:
This line got me looking into file copying in java and now I'm thinking the body of this method could be simpler. Have a look at [this](http://stackoverflow.com/a/32652909) and let me know your thoughts